internal/engine/dockerprune/docker_pruner.go

```diff
@@ -60,6 +62,14 @@ func (dp *DockerPruner) OnChange(ctx context.Context, st store.RStore) {
 	curBuildCount := state.CompletedBuildCount
 	hasDockerBuild := state.HasDockerBuild()
 	nextToBuild := buildcontrol.NextManifestNameToBuild(state)
+	var imgSelectors []container.RefSelector
+	for _, m := range state.Manifests() {
+		for _, iTarg := range m.ImageTargets {
+			sel := iTarg.ConfigurationRef
+			sel.AsNamedOnly()
```

Comment:
this is a no-op?

Reply:
nope, `ConfigurationRef` is actually of type `RefSelector` 😒 but it's irrelevant now that i'm using `DeploymentRef`!